src/metabase/api/field.clj

```diff
@@ -69,7 +69,7 @@
   [ids]
   (when (seq ids)
     (-> (filter mi/can-read? (t2/select Field :id [:in ids]))
-        (t2/hydrate [:table :db] :has_field_values :dimensions :name_field))))
```

Comment:
I wish we didn't have these differences, they're are always confusing (`get-field` still hydrates these properties).